docs/man/cdt-objdump.1.md

```diff
@@ -0,0 +1,262 @@
+---
+date: April 18, 2023
+section: 1
+title: 'CDT-OBJDUMP'
+footer: AntelopeIO
+header: Contract Development Toolkit (CDT)
+---
+
+# NAME
+
+cdt-objdump - manual page for cdt-objdump
+
+# DESCRIPTION
+
+cdt-objdump is a CDT object file dumper
+
+# SYNOPSIS
+ 
+cdt-objdump \[options\] \<input object files\>
+
+# OPTIONS
+
+Generic Options:
+
+**\--help** - Display available options (**\--help-hidden** for more)
+
+**\--help-list** - Display list of available options
+(**\--help-list-hidden** for more)
+
+**\--version** - Display the version of this program
+
+cdt-objdump MachO Specific Options:
+
+**\--arch=**\<string\> - architecture(s) from a Mach-O file to dump
+
+**\--archive-member-offsets** - Print the offset to each archive member
+for Mach-O archives (requires **-macho** and **-archive-headers**)
```

Comment:
In several places `-macho` is used instead of `--macho`. I'm just 
```suggestion
for Mach-O archives (requires **\--macho** and **\--archive-headers**)
```